common/test/common/templates/TemplateServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package common.templates;
+
+import common.pages.PageData;
+import org.junit.Test;
+import play.twirl.api.Html;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TemplateServiceTest {
+
+    @Test
+    public void fillsToHtml() throws Exception {
+        final String expectedHtml = someHtml();
+        final TemplateService templateService = (t, p) -> expectedHtml;
+        final Html html = templateService.fillToHtml(someTemplateName(), somePageData());
+        assertThat(html.body()).isEqualTo(expectedHtml);
+    }
+
+    private String someHtml() {
+        return "<html>" +
+                "<head></head>" +
+                "<body>Hello world!</body" +
+                "</html>";
+    }
+
+    private String someTemplateName() {
```

Comment:
Just a way to hide that I actually don't care about the value. The template service is dummy, it always returns the same no matter what do you pass. I could pass `"foo"` or `null` anywhere, but I thought it was easy to get distracted about the value and people would wonder there is a `null` or `"foo"` there, so I hid it behind a `some...` method.
It's just a little detail I thought it was a good idea, but if you guys think otherwise I don't have a problem removing it :P